plugins/jobs/web_client/stylesheets/jobListWidget.styl

```diff
@@ -79,4 +79,4 @@ ul.g-jobs.nav.nav-tabs
   overflow-x auto
 
 .g-no-job-record
-  text-align center
\ No newline at end of file
+  text-align center
```

Comment:
It should already be on, but only checks Javascript files, not Pug or anything else.